libraries/api/test/auth_test.js

```diff
@@ -320,8 +324,7 @@ suite(testing.suiteName(), function() {
           } else {
             assert(err.message.match(/This request requires Taskcluster/));
           }
-
-          return res.reply({});
+          return res.reportError('InsufficientScopes', 'Insufficient scopes as intended');
```

Comment:
Could this change mean that another test in here that asserts a 403 is becoming a false negative? It worries me a bit to change this

Reply:
Change reverted.